tailwind.config.js

```diff
@@ -7,6 +10,15 @@ module.exports = {
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
   ],
   theme: {
+    fontSize: {
+      "3xs": "0.375rem",
+      "2xs": "0.625rem",
+      ...defaultTheme.fontSize,
+    },
+    screens: {
+      xs: "475px",
+      ...defaultTheme.screens,
+    },
```

Comment:
these custom breakpoints and custom font sizes are NOT needed, you should be able to do everything with built in tailwind, contact me for more information.